stdlib/Random/src/deprecated.jl

```diff
@@ -37,3 +37,6 @@ end
 
 # PR #25668
 @deprecate RandomDevice(unlimited::Bool) RandomDevice(; unlimited=unlimited)
+
+# PR #Current
+@deprecate srand Random.seed
```

Comment:
I believe this should have a `false` at the end to avoid exporting it, i.e.
```julia
@deprecate srand Random.seed false
```

Reply:
I also don't think this form is supported, I think you need to write a custom message with `depwarn(...)`

Reply:
No this works okay:
```
julia> module A
       f() = 1
       Base.@deprecate g A.f false
       end
Main.A

julia> using .A

julia> A.g()
┌ Warning: `g` is deprecated, use `A.f` instead.
│   caller = top-level scope at none:0
└ @ Core none:0
1
```

Reply:
I will update it when I wake up tomorrow. In the meantime, does anyone know where the:
```WARNING: importing deprecated binding Base.Random into Test.
Base.Random is deprecated, run `using Random` instead
ERROR: LoadError: error compiling top-level scope: deprecated binding: Base.Random``` and similar come from? As they seem to be the main culprit to the tests failing.

Reply:
In Test.jl you need `using Random` instead of just `using Random: ...`, which only imports specific items from Random (and not `Random` itself).

Reply:
Got it. Thank you.

Reply:
> you need using Random instead of just using Random: ..., 

In this case, `import Random` would be slightly better I think (imports only the binding `Random` itself), as there is already a line `using Random: ...` with explicit functions. But not worth changing again.
BTW, wouldn't it make sense that `using Random: ...` also imports the `Random` binding itself?